docs/user-guide/SettingsFile.md

```diff
@@ -27,8 +27,16 @@ Checkers' Friendly Names:
 * PayloadBody
 * Examples
 
-### custom_checkers: str (default None)
-Path to a custom checker file that will be loaded during runtime.
+### custom_bug_codes: list(str)
+List of status codes that will be flagged as bugs.
+
```

Comment:
Please provide an example that also shows the regex format (right now that is not documented, as far as I can tell)